core/schema/cache.graphqls

```diff
@@ -2,11 +2,13 @@
 scalar CacheID
 
 extend type Query {
-  "Constructs a cache volume for a given cache key."
+  """
+  Constructs a cache volume for a given cache key.
+  Example:
+  cacheVolume("myapp-cache")
+  """
   cacheVolume(
-    """
-    A string identifier to target this cache volume (e.g. "myapp-cache").
```

Comment:
The `e.g.` was fine here.

Reply:
I put examples on the function call to be more explicit than the field parameter.
See my comments [here](https://github.com/dagger/dagger/pull/4367#issuecomment-1382619290) for details and lmk what you would prefer.